data/coalition/coalition missions.txt

```diff
@@ -1392,3 +1392,207 @@ mission "Arachi Cartoonist 2"
+	to offer
+		has "Coalition: Council of Ahr: active"
+	source "Stronghold of Flugbu"
+	destination "Ahr"
+	on offer
+		conversation
+			`Once your ship has landed, you receive a message from House Bebliss, directing you to a large hotel in the spaceport city. The security guards don't let you in, so you wait for the representative to come out. About half an hour of waiting later, an elevator opens for many Arachi to come out, most of them in formal uniforms and carrying various bags of luggage. There's also an older, and even more formally dressed Arach.`
+			`	"My transport to the Council you are, correct?" She asks, and you confirm. "Splendid. Lady Plubat, I am, Captain. Pleased that transport the will of House Bebliss, such an exquisite captain will. Finally gotten it right, those arranging the transport to the Council have."`
+			`	She asks you to lead the way to your ship, as the Arachi in uniform follow the two of you, carrying her luggage. "Told, I was, about which representatives tasked with transporting, you were. Bebliss excepted, not the most... intriguing company, they are, so grateful that you're transporting at least one proper guest, you should be."`
+			choice
+				`	"Looks like you hold your house in very high regards."`
+					goto house
+				`	"You don't seem very excited for the Council. Is there something wrong with the other houses?"`
+					goto council
+				`	"Is there anything I should know about the hosts of the Council? I heard they were royalty."`
+			`	"The Oberegts? Well, for one, one of the few houses to honor their noble lineage, they are. Elected as a figurehead, in ancient times they were. Never more than that, they have been, lacking real power and focusing only on Ahr, though contributed much to our understanding of medicine, they have. A figurehead, they are, but a good one, and as such some degree of respect, I must grant them."`
+				goto end
+			label house
+			`	"Keep the Coalition together, House Bebliss does. What makes communication from one system to another possible, our efforts are. Other than the obvious importance of our work, one of the few houses that kept their noble bloodline pure, and who kept their original emblems, we are. Prejudice against those the other houses mixed with, I have not, but greatly value tradition, we do, and broken tradition, they have, in marrying commoners or... those of other species."`
+				goto end
+			label council
+			`	"Drifted away from our old ways, most of them have. Take Mallob and Garbarag, barely houses anymore, they are. Fully transitioned into corporations, they have. Claim noble heritage, Debdo does, but in fact a splinter group of our own house, they are, the traitourous offshoots. The worst of them all, however, House Chamgar is. Lunatics clinging on to an ancient rivalry, they are, liars and revisionists who to tarnish our image, wish. Declare war on us, they would, if subjected to the Heliarch government, our houses were not."`
```

Comment:
"Traitorous offshoots" sounds very aggressive. Was there an active betrayal, or were they just a group that split off to pursue slightly different interests, or because their association was unhelpful due to distance/focus/other?
Unless there was an active betrayal, as opposed to a sub-group simply moving on to do their own thing, this phrasing seems excessive - particularly for something that no doubt happened many, many decades or even centuries in the past.

Reply:
It's intentional that both the representatives of both House Bebliss and Chamgar are being dishonest in some of what they're saying, and trying to paint other houses as bad or hype up their house. This is one case of that.

Reply:
Right, but this is a third unrelated house. Obviously Bebliss and Chamgar have an *active* disagreement between each other. What this does is imply that Bebliss and *Debdo* do also have an *active* disagreement ongoing between each other.

Since this is the first time such things are being brought up and Debdo isn't directly included, I'm just checking that this additional implication is intentional.

Reply:
I didn't want to make eight different landing missions, one for each house, probably couldn't even think of original things for them all to say in response to the questions really, so I picked one from each "set" of houses.

If you look at the house emblems, they are divided into three groups, which are meant to reflect part of how the house "behaves":

- Traditional emblems, unchanged from when they first formed. These are those of House Bebliss, Debdo, and the Royal family;
- Emblems that evoke something related to the craft of the houses, being a bit more commercial and modern. These are those of House Ablomab (a drill, for they deal with mining and metallurgy), Idriss (they are supposed to look like circuits, for they deal in electronics and computers), and Plumtab (the box is supposed to be a monitor that shows the stock market on their world, with the coins inside it being in free fall, and some piles of coins at the bottom, below the box);
- Emblems of houses that have fully transitioned into corporations, heavily hinting at their craft as well as their world. These are those of House Chamgar (the circle at the bottom is meant to be Hammer of Debrugt, with the arrows being rockets fired at three Quarg rings), Garbarag (rolling green hills under a shining sun to represent their farms and ranches), and Mallob (their moto is "strong as the mountains, spacious as the skies").

And so I picked a house from each of the three groups, choosing what I saw as the most extreme version of their respective group, with Bebliss being adamant about tradition and its position as nobility, Ablomab struggling with the workload they have to bear as they are those to produce the alloys needed for an incredible portion of Coalition hulls, and Chamgar trying to sell itself as some legendary group and not caring to talk about much else.

Reply:
Right, that's cool and interesting, but wouldn't that make Bebliss and Debdo likely to be friends?

Reply:
By "how a house behaves" I mean their general mannerisms and views on traditional Arach nobility practices, not their standings on the other houses or what houses they'd be friends with (for example, every single house has positive relations with the Royal family, House Garbarag in particular has a very long alliance with Ahr's rulers, providing a lot of food for the planet in exchange for technology, having goods manufactured for them, advertisement among the masses on the capital), and not their views on the Heliarch government either.
House Bebliss started out as a spyguild, clashing constantly with House Chamgar who were expansionists and developed a lot of new technology (namely the repair modules); their conflicts would eventually have House Debdo form as offshoots from Bebliss, seeking to use the spyguild knowledge in other ways, and using that knowledge to ally with some other houses to first establish themselves, the big alliance being them selling off Bebliss secrets to House Chamgar in exchange for some of their technology.

I feel the missions already have enough exposition dialogue as is, though, so don't think it's worth detailing such things.

Reply:
Ah yes, that's a solid backdrop for that opinion, and a good space to be expanded on later then. Cool.